docs/multicluster/antctl.md

```diff
@@ -47,6 +51,17 @@ anctcl mc create membertoken NAME -n NAMESPACE [-o OUTPUT_FILE]
 
 To see the usage examples of these commands, you may also run `antctl mc create [subcommand] --help`.
 
+## antctl mc delete
+
+`antctl mc delete` command can delete a token of a member cluster. The command will also delete the 
```

Comment:
"can delete" -> "deletes"

Probably "a member token", as a token can be used by multiple clusters.